.github/workflows/release-candidate.yml

```diff
@@ -56,7 +56,7 @@ jobs:
         uses: actions/setup-java@a7ab372554b6eb1a8eb25e7d9aec1cc9f3ea1a76 # v4.6.0
         with:
           distribution: temurin
-          java-version: 8
+          java-version: 17
```

Comment:
Note that we'll also need to update the [build release container](https://github.com/apache/daffodil-infrastructure/tree/main/containers/build-release) to use Java 17. Should just be a one-line change here:

https://github.com/apache/daffodil-infrastructure/blob/main/containers/build-release/src/daffodil-build-release#L108